src/Viewer.js

```diff
@@ -55,20 +53,46 @@ export default class Viewer extends Component {
   };
 
   shuffle = () => {
-    this.setState({ pos: 0, q: shuffled(this.state.q) });
+    this.setState({ pos: 0, q: _.shuffle(this.state.q) });
   };
 
+  toggle = () => {
+    this.setState({ dropdown: !this.state.dropdown });
+  }
+
+  select = (event) => {
+    const selected = event.target.innerText
+    this.setState({
+        currentType: selected,
+        q: _.shuffle(this.props.files[selected]),
+        pos: 0
+    })
+}
+
   render() {
-    const { q, pos } = this.state;
+    const { q, pos, dropdown, currentType } = this.state;
+    const { types } =  this.props
+    let dropItems = []
+    for (let key in types) {
+      dropItems.push(<DropdownItem key={key} onClick={this.select}>{types[key]}</DropdownItem>)
+    }
     return (
       <div className="viewer-container">
         <div className="button-container">
           <button disabled={pos === 0} onClick={this.prev}>
-            Previous
+            <FaCaretLeft size={40} color='#f8f9fa'/>
           </button>
           <button onClick={this.shuffle}>Shuffle</button>
+          <ButtonDropdown isOpen={dropdown} toggle={this.toggle}>
+            <DropdownToggle caret color="black">
+              {currentType}
+            </DropdownToggle>
+            <DropdownMenu>
+              {dropItems}
+            </DropdownMenu>
+          </ButtonDropdown>
```

Comment:
The styling for the dropdown looks different than the other buttons, wondering if you caught this:

![image](https://user-images.githubusercontent.com/45303840/49016054-ea944d00-f152-11e8-85dd-8bb6865508f8.png)